activities/tool-management/voys.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# Voys
+
+Voys is our VoIP provider.
+
+## Using Voys
+
+Employees that need to call regularly receive an account. A local (020) number will be assigned. Voys can be used with an app on a mobile phone. There is [documentation available](https://www.voys.nl/hulp/toestellen/) (in Dutch, use Chrome to translate it) to help with setting it up, both for administrators as well as users.
```

Comment:
```suggestion
Employees that need to call regularly receive an account. A local (020) number will be assigned. Voys can be used with an app on a mobile phone. There is [documentation available](https://www.voys.nl/hulp/toestellen/) (in Dutch) to help with setting it up, both for administrators as well as users.
```
I've removed the advice to use a specific translation tool, since there are lots available.